code/game/objects/items/miscellaneous.dm

```diff
@@ -12,6 +12,62 @@
 	w_class = WEIGHT_CLASS_SMALL
 	attack_verb = list("warned", "cautioned", "smashed")
 
+/obj/item/herobeacon
+	name = "heroic beacon"
+	desc = "To summon heroes from the past to protect the future."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-blue"
+	item_state = "radio"
+	var/static/list/display_names = list()
+
+/obj/item/herobeacon/attack_self(mob/user)
+	if(user.canUseTopic(src, NO_TK))
```

Comment:
NO_TK doesn't do what you think it does here. canusetopic args are (user, be_close, no_dexterity, no_tk), with the capitalized BE_CLOSE, NO_DEXTERY, and NO_TK things just being defines for true so it's easily readable.

just remove this here and handle it at the start of beacon_hero instead.